generators/client/templates/angular/webpack/webpack.custom.js.ejs

```diff
@@ -0,0 +1,108 @@
+const webpack = require('webpack');
+<%_ if (enableTranslation) { _%>
+const MergeJsonWebpackPlugin = require("merge-jsons-webpack-plugin");
+<%_ } _%>
+const SimpleProgressWebpackPlugin = require('simple-progress-webpack-plugin');
+const FriendlyErrorsWebpackPlugin = require('friendly-errors-webpack-plugin');
+const BrowserSyncPlugin = require('browser-sync-webpack-plugin');
+const BundleAnalyzerPlugin = require('webpack-bundle-analyzer').BundleAnalyzerPlugin;
+
+const tls = process.env.TLS;
+
+module.exports = (config, options) => {
+  // RULES
+  if (config.mode === 'development') {
+    config.module.rules.push({
+      test: /\.(j|t)s$/,
+      enforce: 'pre',
+      loader: 'eslint-loader',
+      exclude: /node_modules/
+    });
+  }
```

Comment:
`eslint-loader` is deprecated and can be replaced with https://github.com/webpack-contrib/eslint-webpack-plugin (this can be also separate PR)

Reply:
I think i'll do another PR for this :)